venter/tests.py

```diff
@@ -66,10 +66,24 @@ def test_complaint(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data['description'], 'test')
 
-        response = self.client.put(url, data, format='json')
-        self.assertEqual(response.status_code, 200)
+        url = '/api/venter/complaints/'+cid+'/upvote'
```

Comment:
Add spacing before and after `+`. A cleaner way would be to use the `%` operator